packages/SwingSet/src/devices/mailbox/mailbox.js

```diff
@@ -94,6 +94,10 @@ export function exportMailbox(inout) {
   };
 }
 
+/**
+ *
+ * @param {Map<*, *>} state
+ */
```

Comment:
```suggestion
/** @param {Map<*, *>} state */
```

is that type not inferred?

Reply:
good point, this doesn't add anything. removed.